src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/best-practices/opentelemetry-best-practices-resources.mdx

```diff
@@ -8,21 +8,117 @@ metaDescription: Here are some tips for working with OpenTelemetry resources and
 freshnessValidatedDate: never
 ---
 
-A resource in OpenTelemetry represents information about an entity generating telemetry data. Make sure all telemetry data sent to New Relic is associated with a resource so it can be linked with the appropriate entity in New Relic. The [OpenTelemetry Resource SDK specification](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/resource/sdk.md) defines the functionality implemented by all language SDKs for defining a resource.
+All data from OpenTelemetry is associated with a [resource](https://opentelemetry.io/docs/concepts/resources).
```

Comment:
Consider renaming the name of this page from "OpenTelemetry resources: Best practices". Maybe just "OpenTelemetry Resources"?

Reply:
I just left it as is since all the pages that are siblings to this one has the awkward "Best practices" subtitle. May be we just fix them all at once when we get everything moved to where we ultimately want it.